examples/image_classification.ipynb

```diff
@@ -0,0 +1,5278 @@
+{
```

Comment:
<p>Would suggest updating to:</p><p> </p><pre class="ql-syntax" spellcheck="false">%%capture
! pip install git+https://github.com/huggingface/huggingface_hub.git@main
! sudo apt -qq install git-lfs
! git config --global credential.helper store
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>